packages/launch/src/focused-launch/index.tsx

```diff
@@ -23,12 +23,14 @@ function noop( ...args: unknown[] ) {
 }
 
 const FocusedLaunch: React.FunctionComponent = () => {
-	const { getSite } = useSelect( ( select ) => select( 'core' ) ) as any;
-	const title = getSite()?.title;
-
 	const site = useSite();
 
-	const [ siteTitle, setSiteTitle ] = React.useState( title );
+	const siteTitle = useSelect( ( select ) =>
+		select( SITE_STORE ).getSiteTitle( window._currentSiteId )
```

Comment:
Just a note: `window._currentSiteId` won't work in [Calypso integration](https://github.com/Automattic/wp-calypso/pull/46834). Probably worth adding a comment.